framework/src/org/apache/cordova/CordovaWebViewClient.java

```diff
@@ -171,8 +171,13 @@ public void onPageStarted(WebView view, String url, Bitmap favicon) {
     public void onPageFinished(WebView view, String url) {
         super.onPageFinished(view, url);
         // Ignore excessive calls.
-        if (!isCurrentlyLoading) {
-            return;
+        //     On KitKat, loading about:blank does not invoke WebViewClient#onPageStarted
+        //     So do not check isCurrentLoading if app is running on KitKat
+        //         https://code.google.com/p/android/issues/detail?id=65701
+        if (android.os.Build.VERSION.SDK_INT != android.os.Build.VERSION_CODES.KITKAT){
```

Comment:
You don't need to re-request a PR, but you do need to make the change an upload it to github (either as a separate commit, or by force-pushing an ammended commit). Then, add a comment to let me know you've done so and I'll pull it in (once you do it, it should show up on this interface).